### PR TITLE
Use the new MPP model

### PR DESCRIPTION
**Pull request by raniejade**
PR is not as big as you think, it's mostly gradle changes. The file moves makes it look big. So far I've tested the `jvm` and `mpp` samples - and they all work. Docs may need to be updated, but I'll work on that on a separate PR.

### CODE REVIEW

File: samples/jvm/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
-    ext.kotlin_version = '1.3.0'
-    ext.spek_version = '2.0.0-rc.1'
+    ext.kotlin_version = '1.3.11'
+    ext.spek_version = '2.0.0-rc.1.181+20190113T062549Z'
```
Comment (raniejade) [inline samples/jvm/build.gradle:3]: todo: use proper version when published.

File: samples/multiplatform/build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 buildscript {
-    ext.kotlin_version = '1.3.0'
-    ext.spek_version = '2.0.0-rc.1'
+    ext.kotlin_version = '1.3.11'
+    ext.spek_version = '2.0.0-rc.1.181+20190113T062549Z'
```
Comment (raniejade) [inline samples/multiplatform/build.gradle:3]: todo: use proper version when published.

File: build.gradle [hunk 3]
```diff
@@ -32,8 +32,9 @@ afterEvaluate {
 
 
 allprojects {
-    // This causes a circular dependency for some reason.
-    // group = 'org.spekframework.spek2'
+    // Previously, this causes a circular dependency for some reason.
+    // It now works.
+     group = 'org.spekframework.spek2'
```
Comment (vanniktech) [inline build.gradle:36]: Spacing police 🚓

File: settings.gradle [hunk 4]
```diff
@@ -5,30 +5,35 @@ pluginManagement {
...
+include 'spek-dsl'
 
-include 'spek-runtime:common'
-include 'spek-runtime:jvm'
+include 'spek-runtime'
 
 include 'spek-runner:junit5'
 
+include 'integration-test'
+//include 'integration-test:common'
```
Comment (vanniktech) [inline settings.gradle]: Remove?

**Review by vanniktech — APPROVED** [review]

**Review by vanniktech — APPROVED** [review]

**Pull request merged**